modules/RelationalDataLoader.py

```diff
@@ -58,31 +65,40 @@ def raw_connection_string_to_valid_source_connection_string(self, connection_str
     def get_arguments(self):
         parser = argparse.ArgumentParser(description='Relational Data Loader')
 
-        parser.add_argument('source-connection-string', metavar='source-connection-string',
-                            type=self.raw_connection_string_to_valid_source_connection_string,
-                            help='The source connections string. Eg: mssql+pyodbc://dwsource or '
-                                 'csv://c://some//Path//To//Csv//Files//')
-
-        parser.add_argument('destination-engine', metavar='destination-engine',
-                            help='The destination engine. Eg: postgresql+psycopg2://postgres:xxxx@localhost/dest_dw')
-
         parser.add_argument(
-            'configuration-folder',
-            metavar='configuration-folder',
-            help='The configuration folder. Eg C:\\_dev\\oscars-misc\\el-pipeline-spike\\configuration\\')
-
-        parser.add_argument('--log-level',
+            'source_connection_string',
+            metavar='source-connection-string',
+            type=self.raw_connection_string_to_valid_source_connection_string,
+            help='The source connections string as a 64bit ODBC system dsn. Eg: mssql+pyodbc://dwsource or '
+            'csv://c://some//Path//To//Csv//Files//')
+
+        parser.add_argument('destination_connection_string',
+                            metavar='destination-connection-string',
+                            help='The destination database connection string. Provide in PostgreSQL + Psycopg format. '
+                                 'Eg: \'postgresql+psycopg2://username:password@host:port/dbname\'')
+
+        parser.add_argument('configuration_folder',
+                            metavar='configuration-folder',
+                            help='Absolute or relative path to the models. '
+                                 'Eg \'./models\', \'C:/path/to/models\'')
+
+        parser.add_argument('-f',
+                            '--force-full-refresh-models',
+                            nargs='?',
```

Comment:
Should we add a default value of `*` instead of making the user specify `*` to force a full refresh of all models?

Reply:
Good idea.. done!